tests/elb/test_elb.py

```diff
@@ -189,4 +195,26 @@ def test_elb_add_listener_policy(mock_get_properties, mock_boto3_session):
 
     elb = SpinnakerELB(app='myapp', env='dev', region='us-east-1')
     elb.add_listener_policy(json.dumps(json_data))
-    assert client.set_load_balancer_policies_of_listener.called
+    client.set_load_balancer_policies_of_listener.assert_called_with(LoadBalancerName='myapp', LoadBalancerPort=80, PolicyNames=['policy_name'])
+
+@mock.patch('foremast.elb.create_elb.boto3.session.Session')
+@mock.patch('foremast.elb.create_elb.get_properties')
+def test_elb_add_backend_policy(mock_get_properties, mock_boto3_session):
+
+    json_data = {
+       'job': [
+            {
+                'listeners': [
+                    {
+                        'internalPort': 80,
+                        'backendPolicies': ['policy_name']
+                    },
+                ],
+            }
+       ],
+    }
+    client = mock_boto3_session.return_value.client.return_value
+
+    elb = SpinnakerELB(app='myapp', env='dev', region='us-east-1')
+    elb.add_backend_policy(json.dumps(json_data))
+    client.set_load_balancer_policies_for_backend_server.assert_called_with(InstancePort=80, LoadBalancerName='myapp', PolicyNames=['policy_name'])
```

Comment:
Try not to hard code the values. Reference the input data to avoid code duplication or drift.
```python
test_listener = json_data['job'][0]['listeners'][0]
...assert_called_with(InstancePort=test_listener['internalPort'], ...)
```